pkg/cmd/pulumi/new.go

```diff
@@ -768,6 +769,8 @@ func chooseTemplate(templates []workspace.Template, opts display.Options) (works
 		if height > len(options) {
 			height = len(options)
 		}
+
+		height = height - buffer
```

Comment:
If we change this height to a much smaller value, is the flickering as severe?

Reply:
Don't think there's a clear maximum, even something as small as 10 on a max sized terminal causes pretty bad flickering on my mac default terminal